src/org/usfirst/frc/team694/robot/commands/auton/LiftMoveUpHeightCommand.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team694.robot.commands.auton;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class LiftMoveUpHeightCommand extends Command {
+    
+    public double height;
+
+    public LiftMoveUpHeightCommand(double height) {
+        requires(Robot.lift);
+        this.height = height;
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+        Robot.lift.resetEncoders();
+        Robot.lift.setBrakeOff();
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.setLiftMotors(1);
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return Robot.lift.getLiftHeight() >= height;
+    }
+
+    // Called once after isFinished returns true
+    protected void end() {
+        Robot.lift.setBrakeOn();
```

Comment:
You also want to stop the lift! What you're doing here is you're applying the brake on the lift while the motor is still running, and that might stall the motors. You must also stop them.